database/engine/global_uuid_map/global_uuid_map.c

```diff
@@ -27,11 +42,15 @@ int guid_store(uuid_t uuid, char *object)
     if (PPJERR == PValue)
         fatal("JudyHSIns() fatal error.");
     if (*PValue == NULL) {
-        uuid_t *value = mallocz(sizeof(uuid_t));
-        memcpy(value, &uuid, sizeof(uuid_t));
+        uuid_t *value = (uuid_t *) mallocz(sizeof(uuid_t));
+        uuid_copy(*value, uuid);
```

Comment:
This is less clear than the old code because a `uuid_t` is not really a value but a pointer.